Library/Formula/hugo.rb

```diff
@@ -186,7 +178,7 @@ def install
 
     system "go", "build", "-o", bin/"hugo", "main.go"
 
-    system bin/"hugo", "genautocomplete", "--completionfile=#{buildpath}/hugo.sh"
+    system bin/"hugo", "gen", "autocomplete", "--completionfile=#{buildpath}/hugo.sh"
```

Comment:
The release notes say `hugo gen` can create manpages too. Could you add a second command for that? Thanks so much!